ommx_da4_adapter/adapter.py

```diff
@@ -201,7 +201,7 @@ def _generate_binary_polynomial(self) -> BinaryPolynomial:
 
         binary_polynomial_terms = [
             BinaryPolynomialTerm(
-                c=value, p=self._replace_polynomials_with_variable_map(key)
+                c=value, p=self._replace_polynomials_with_variable_map(tuple(key))
```

Comment:
```suggestion
                c=value, p=self._replace_polynomials_with_variable_map(key)
```

ommx v2.0.0b3でtermsから得られる辞書のキーはtuple型に変更されたので、tupleへの変換は不要になっていますね。